docs/TESTS.md

```diff
@@ -24,6 +24,53 @@ Working through each exercise is a process of:
  * Refactor your implementation to enhance readability, reduce duplication, etc.
  * Uncomment the next test
 
+<<<<<<< Updated upstream
+=======
```

Comment:
This is due to a conflict in the merge request, and this wasn't removed during the 'review' locally.